src/type_object.rs

```diff
@@ -315,6 +309,13 @@ where
 
     type_object.tp_base = base_type_object;
 
+    let name = match module_name {
+        Some(module_name) => format!("{}.{}", module_name, T::NAME),
+        None => T::NAME.to_string(),
+    };
+    let name = CString::new(name).expect("Module name/type name must not contain NUL byte");
```

Comment:
*NULL ?

Reply:
[NUL is valid in that context](https://en.wikipedia.org/wiki/Null_character) :wink: